### DISCUSSION TITLE
Installation

### DISCUSSION THREAD

**Discussion by notFalkon** [aleksey-hoffman/sigma-file-manager#114] [Q&A] [answered]
I'm almost definitely being an idiot, but how do I install this? I downloaded the zipped folder and extracted it, and now I have no idea what to do.
![image](https://user-images.githubusercontent.com/79824068/124022850-07254f80-d9f6-11eb-8dd2-e11a209df8a0.png)

**Comment by aleksey-hoffman** [comment, marked as answer]
You downloaded the source code of the app, mate.
If you want the installable file, you can download it on the [releases](https://github.com/aleksey-hoffman/sigma-file-manager/releases) page

**Reply by notFalkon** [reply]
Oh thanks
